docs/en/quick-answers/administering/where-is-my-cache.md

```diff
@@ -60,6 +60,6 @@ If your configuration uses a shared core, then this will be located inside your
 
 ## Thumbnails
  
-Thumbnails used by Toolkit apps (like the [Loader](https://support.shotgunsoftware.com/entries/95442527)) are stored in the local Toolkit cache. They are stored per Project, Pipeline Configuration, and App (as needed). The structure beneath the root cache directory is as follows:
+Thumbnails used by Toolkit apps (like the [Loader](hhttps://developer.shotgridsoftware.com/a4c0a4f1/?title=Loader)) are stored in the local Toolkit cache. They are stored per Project, Pipeline Configuration, and App (as needed). The structure beneath the root cache directory is as follows:
```

Comment:
hhttps probably wrong here, and there's an extra ) on the end of the URL

Reply:
Thanks! Spotted a couple more and cleaned these up as well.